test/integration/driver.go

```diff
@@ -59,7 +59,7 @@ const (
 	caImage = "gcr.io/istio-testing/istio-ca:2baec6baacecbd516ea0880573b6fc3cd5736739"
 
 	// mixerImage specifies the default mixer docker image used for e2e testing *update manually*
-	mixerImage = "gcr.io/istio-testing/mixer:986c6e1eb9c7dcff42e3df5aefa9ce858ece885e"
+	mixerImage = "gcr.io/istio-testing/mixer:36994fb05d97abfa44761bafa5d1efec8d132b47"
```

Comment:
at some point, i feel like this needs to be the docker hub image...

Reply:
Once mixer master is gated by istio/istio tip e2e test, we can switch to using tip of mixer.